leakcanary-analyzer/src/main/java/leakcanary/Serializables.kt

```diff
@@ -47,6 +48,7 @@ object Serializables {
     }
   }
 
+  @Suppress("UNCHECKED_CAST")
```

Comment:
Both for the `T?` cast

Reply:
In that case, consider reifying and doing `if (object is T) object as T else null`

Reply:
Ah yeah that'd be better 👍